### ISSUE TITLE
SecondOrderWrapperComponent parameter on withMappedNavigationParamsDecorator needs to be optional

### ISSUE THREAD

**Issue by FutureGUIs** [vonovak/react-navigation-props-mapper#13]
`SecondOrderWrapperComponent` needs to be optional, marked with a `?` in the type definition at least, correct?

```
 interface withMappedNavigationParamsDecorator {
    (SecondOrderWrapperComponent?: React.ReactNode): WrappedComponent;
```

**Comment by vonovak** [comment]
hi! yes, the param is optional, sorry, the TS typings are probably incorrect, they also do not include `wrappedComponent` - I do not currently use this in a TS project hence I did not fix them. a PR is welcome

**Comment by vonovak** [comment]
fix released in 1.0.1
